src/erlpool_compile.erl

```diff
@@ -4,21 +4,28 @@
 
 -export([compile_settings/1]).
 
+-include_lib("syntax_tools/include/merl.hrl").
+
 compile_settings(SettingsList) ->
-    case merl:compile_and_load(get_settings_code(SettingsList)) of
+    Forms = get_settings_code(SettingsList),
+    file:write_file("/tmp/erlpool_globals_gen.erl", erl_prettypr:format(erl_syntax:form_list(Forms))),
```

Comment:
What is the point of this line ?

Reply:
Sorry, it's just for debugging, will comment it later

Reply:
Thanks. remove that line and also make sure you rebase with my last fix regarding atoms. Then I will merge

Reply:
I've merged your master branch, result of common test looks good